spec/models/user_spec.rb

```diff
@@ -2645,4 +2645,37 @@ def reset_last_seen_cache!(user)
       expect(result).to be(true)
     end
   end
+
+  describe "#whisperer?" do
+    before do
+      SiteSetting.enable_whispers = true
+    end
+
+    it 'admin is whisperer' do
```

Comment:
```suggestion
    it 'returns true for an admin user' do
```

Do go through the all the descriptions because this is supposed to read like a conversation. See https://relishapp.com/rspec/rspec-core/v/3-8/docs/example-groups/basic-structure-describe-it